serverlessworkflow/modules/ROOT/pages/cloud/operator/enabling-jobs-service.adoc

```diff
@@ -0,0 +1,170 @@
+namespace configuration via the `SonataFlowPlatform` CR. With this approach, the Operator will take care of configuring the
+Jobs Service instance and ensure it is in sync with the specification in the CR. When the operator manages the Jobs Service lifecycle,
+it will inject properties in `SonataFlow` workflows at creation time to enable the workflows to communicate with the Jobs Service
+It is possible to deploy the Jobs Service manually, but the Operator also provides a convenient way to combine it with the namespace configuration via the `SonataFlowPlatform` CR. This approach allows the Operator to configure the Jobs Service and make sure it is in line with the Specification in CR. When the Operator manages the jobs service lifecycle, it injects properties at creation time in the `SonataFlow` workflows to communicate with the Jobs Service during execution, eliminating the need to include these properties in the workflow CR instance of the `SonataFlow` workflow.
+
+== Configuring Jobs Service in the SonataFlowPlatformCR
+
+To enable the deployment of a Jobs Service instance, the `SonataFlowPlatform` CRD exposes a set of fields that allow the user to 
+configure the running instance. 
+
+==== Ephemeral persistence
+The basic runtime is to deploy the Jobs Service with an ephemeral backend running in the same container
+as the Jobs Service runtime.
+
+[source,yaml,subs="attributes+"]
+---
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  services:
+    jobService: {}
+---
+
+When executing this manifest, the operator will reconcile generating a pod hosting the Jobs Service:
+
+[source,shell,subs="attributes+"]
+---
+$>kubectl get pod -n sonataflow
+NAME                                               READY   STATUS    RESTARTS   AGE
+sonataflow-platform-jobs-service-cdf85d969-sbwkj   1/1     Running   0          108s
+---
+
+Keep in mind that this setup is not recommended for production environments, especially because the data does not persist when the pod restarts.
+
+==== Using an existing postgreSQL service
+For robust environments, it is recommended to use a dedicated database service and configure Jobs Service to make use of it. Currently, the Jobs Service
+only supports PostgreSQL database.
+
+Configuring Jobs Service to communicate with an existing PostgreSQL instance is supported in two ways. In both cases it requires providing the persistence
+configuration, one by using the Jobs Service's persistence field and the other one using the persistence field defined in the `SonataFlowPlatform` CR
+deployed in the same namespace.
+
+By default, the persistence specification defined in the `SonataFlow` workflow's CR takes priority over the one in the `SonataFlowPlatform` persistence specification.
+
+===== Using the persistence field defined in the `SonataFlowPlatform` CR
```

Comment:
Depends on how you want to structure the documentation. Here I am taking the assumption that this document contains all relevant information on how configure the job service, including persistence. If you prefer to contain persistence in its own scope, then I would suggest to move it not in the workflow-persistence.adoc that you mention, but in a `platform-persistence.adoc` or `persistence.adoc` where it contains how to configure persistence for workflows and platform services.

Let me know which way you prefer.

Reply:
Summon @domhanak since he's restructuring a few things here. I think we can move to platform-persistence.adoc, would make more sense. Thank you for the suggestions!